Sources/DynamicNotchKit/DynamicNotch.swift

```diff
@@ -37,8 +37,12 @@ public class DynamicNotch<Content>: ObservableObject where Content: View {
         case auto
     }
 
-    // Animation
-    private var maxAnimationDuration: Double = 0.8 // This is a timer to deinit the window after closing
```

Comment:
I guess this should have been more descriptive, but this comes from the below code:
```swift
var animation: Animation {
     if #available(macOS 14.0, *), notchStyle == .notch {
         Animation.spring(.bouncy(duration: 0.4))
    } else {
        Animation.timingCurve(0.16, 1, 0.3, 1, duration: 0.7)
    }
}
```

Since these animations vary in duration, I chose 0.8 so that we can be sure that the notch has been fully shrunk before de-initializing the window.

So, I don't think it makes sense to create an `AnimationDuration` enum.

Reply:
Reverted. I thought of having more options to hide the whole notch view immediately without seeing this small notch at the end of animating.
![1](https://github.com/user-attachments/assets/1f4ac12a-17b3-4a15-b89e-0015d7df74c8)

Reply:
Yes, this is a focus ring. Since it probably doesn't make sense to have those inside the notch, it would make sense to add `.focusable(false)` at the end of `NotchView` and `NotchlessView`. Would you be able to do this?

Reply:
I tried adding `.focusable(false)` at the end of `NotchView` and `NotchlessView` but it won't work.

Directly apply to `TextField` on a custom view is working:
```swift
TextField("Enter search here...", text: $text)
    .textFieldStyle(.plain)
    .focusable(false)
    .foregroundStyle(.white)
    .padding(4)
    .border(.red, width: 1)
```

Reply:
Ah alright, in that case, please do that. Otherwise, this seems ready to be merged!